test/new-e2e/system-probe/test-runner/files/no_tracer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pkg/network/tracer": {
+        "run": "TestUSMSuite"
```

Comment:
What is the duration of `TestUSMSuite`? I'm wondering if it is worth the extra complexity to skip/run individual tests, rather than just dealing at the package level.

Reply:
This might need to be a regex to ensure no other tests are accidentally included.

Reply:
> What is the duration of TestUSMSuite? 
It will have the same duration as `pkg/network/tracer`

> I'm wondering if it is worth the extra complexity to skip/run individual tests, rather than just dealing at the package level.
It does have added complexity, I agree. However, this way we don't have to rely on refactors to properly distribute tests across vmsets. Secondly, it also gives finer control of which sets should run the tests.